src/app/components/StatsWidget.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import { useAppSelector, useTranslations } from "hooks";
+import { fetchKpiData, KpiData } from "kpis/kpis-utils";
+import { useEffect, useState } from "react";
+import { simpleFormatNumber } from "utils/simpleFormatNumber";
+import AnimatedCounter from "./AnimatedCounter";
+import { DexterButton } from "./DexterButton";
+
+const StatsWidget = () => {
+  const t = useTranslations();
+  // google sheet data
+  const [kpiData, setKpiData] = useState<KpiData | null>(null);
+  // alphadex data
+  const { pairsList } = useAppSelector((state) => state.rewardSlice);
+
+  // fetch socials & trade volumes (in XRD / USD)
+  useEffect(() => {
+    (async () => {
+      try {
+        const data = await fetchKpiData();
+        setKpiData(data);
+      } catch (error) {
+        console.error("StatsWidget -> fetchKpiData error", error);
+      }
+    })();
+  }, []);
+
+  // TODO: compute actual users with a script fetching all orders on adex.
+  // 1. google sheet data
+  // 1.1. socials data
+  const socials = kpiData?.socials;
+  const numOfIgFollowers = socials?.instagramFollowers ?? 0;
+  const numOfXFollowers = socials?.twitterFollowers ?? 0;
+  const numOfUTubeFollowers = socials?.youtubeSubscribers ?? 0;
+  const totalNumOfFollowers =
+    numOfIgFollowers + numOfXFollowers + numOfUTubeFollowers;
+
+  // 1.2. trade volume data - total & weekly - XRD & USD
+  const tradeVolume = kpiData?.tradeVolume;
+
+  // total
+  const tradeVolumeTotalXrd = tradeVolume?.total?.XRD ?? 0;
+  const tradeVolumeTotalUsd = tradeVolume?.total?.USD ?? 0;
+
+  // weekly
+  const tradeVolumeWeeklyXrd = tradeVolume?.weekly?.XRD?.at(-1)?.value ?? 0;
```

Comment:
Lets skip XRD values and only use USD volume

Reply:
Ok